cumulusci/utils/__init__.py

```diff
@@ -30,7 +30,12 @@
 API_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
 DATETIME_LEN = len("2018-08-07T16:00:56.000")
 
-BREW_UPDATE_CMD = "brew upgrade cumulusci"
+BREW_DEPRECATION_MSG = (
+    "It looks like you have installed CumulusCI using brew."
+    "This method of installation is no longer supported."
+    "Please use the following to install CumulusCI with pipx:\n"
+    "brew install pipx\npipx ensurepath\npipx install cumulusci"
```

Comment:
I forgot that we should add an uninstallation step to get rid of the Homebrew installation: `brew uninstall cumulusci`.

Reply:
done